app/src/main/java/com/github/sdp/mediato/ui/viewmodel/MyProfileViewModel.java

```diff
@@ -35,7 +35,7 @@ public void addReviewToCollection(@NonNull Review review, String collectionName)
      * no changes if such a collection already exists.
      *
      * @param collectionName name of the collection to add
-     * @return true if the collection was sucessfully added, false otherwise
+     * @return true if the collection was successfully added, false otherwise
```

Comment:
Why do we return booleans instead of throwing exceptions ?

Reply:
Particularly if the first statement (`Preconditions...`) might still throw exceptions.

Reply:
Just for convenience where the function was used, but I changed it.